src/src/style.css

```diff
@@ -0,0 +1,79 @@
+:root {
+  --default-color-link: #000;
+  --color-active-link: #00acdc;
+  --background-header: #fff;
+  --box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  background: var(--background-header);
+  box-shadow: var(--box-shadow);
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  display: flex;
+  margin-right: 20px;
```

Comment:
You've added a margin to the right of each .nav-item element. This can lead to unwanted space on the right side of the navigation bar. It's usually better to use the `:not(:last-child)` pseudo-class to add margin to all elements except the last one. This way, you won't have any extra space on the right side of the navigation bar. Here's how it can be done:
css
.nav-item:not(:last-child) {
  margin-right: 20px;
}
```